examples/templateservicebroker/apiserver-config.yaml

```diff
@@ -0,0 +1,4 @@
+kind: TemplateServiceBrokerConfig
+apiVersion: config.templateservicebroker.openshift.io/v1
+templateNamespaces:
+- openshift
```

Comment:
We can define/carry this in openshift-ansible. If it's not of value to anyone else lets just deal with it there.